rfc.xml

```diff
@@ -435,34 +450,55 @@
+			<t>
+				The keylist subscription functionality defined in this
+				document provides a number of security benefits, including:
+			</t>
+			<t>
+				<list style="symbols">
+					<t>
+						The ability for new keys to be quickly distributed
+						across an organization.
+					</t>
+					<t>
+						Removing the complexity of key distribution from
+						end users, allowing them to focus on the content of
+						their communications rather than on key management.
+					</t>
+					<t>
+						The ability for an organization to prevent the
+						spread of falsely attributed keys by centralizing
+						the public key discovery process within their
+						organization without centralized public key hosting.
+					</t>
+					<!-- TODO: others? ~MM -->
+				</list>
+			</t>
+		</section>
+		<section anchor="comparison" title="Relation to Other Technologies">
+			<section anchor="wkd" title="Web Key Directories">
 				<t>
-					The keylist subscription functionality defined in this
-					document provide a number of security benefits, including:
+					Unlike Web Key Directories, keylists are not domain specific. A keylist
+					might contain public key fingerprints for email addresses across several
+					different domains. Moreover, keylists only provide references to public
+					keys by way of fingerprints; Web Key Directories provide the public keys
+					themselves.
 				</t>
+			</section>
+			<section anchor="dns" title="OPENPGPKEY DNS Records">
 				<t>
-					<list style="symbols">
-						<t>
-							The ability for new keys to be quickly distributed
-							across an organization.
-						</t>
-						<t>
-							It removes the complexity of key distribution from
-							end users, allowing them to focus on the content of
-							their communications rather than on key management.
-						</t>
-						<t>
-							The ability for an organization to prevent the
-							spread of falsely attributed keys by centralizing
-							the public key discovery process within their
-							organization.
-						</t>
-						<!-- TODO: others? ~MM -->
-					</list>
+					 A keylist MAY reference public keys corresponding to email addresses
+					 across several different domains. Because managing OPENPGPKEY DNS Records
+					 <xref target="RFC7929" /> for a particular domain requires control of that
+					 domain, the OPENPGPKEY DNS Record system is not suitable for cases in
+					 which keys are strewn about several different domains, including ones
+					 outside of the control of an organization's system adminitrators.
 				</t>
 			</section>
-			<section anchor="securitydrawbacks" title="Security Drawbacks">
+			<!-- We'll want to clean this up, but the messaging we'd like to convey is there. -->
+		</section>
+		<section anchor="securityconsiderations" title="Security Considerations">
 				<t>
 					There is a situation in which keylist subscriptions could
 					pose a potential security threat. If the both the authority
```

Comment:
Found an old typo. "If the both the" should be "If both the"

Reply:
Fixed!